samples/todo-app/frontend/package.json

```diff
@@ -15,7 +15,7 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "request": "^2.88.2",
-        "servicebus": "^2.3.0",
+        "@servicebus/rabbitbus": "^1.0.14",
```

Comment:
💡☕️ This looks good, just mentioning for the archival reasons and folks might be curious:

* THe actual package which is old package was last released 4 years back : https://www.npmjs.com/package/servicebus?activeTab=versions 

#### How `rabbitbus` is a better choice?

Because it is recommended by the authors of the `servicebus` https://github.com/mateodelnorte/servicebus/issues/179

Reply:
Tats, Thank you. You want me to add a comment why this dependency is replaced ?

Reply:
☕️✅ Sounds good, if I May recommend- leave this comment unresolved, for any future reader. Commit history should have it. Or might be to be succinct ass commented link of above tool recommendations as a reference.

Reply:
I am leaving this comment unresolved for historical purposes.

Reply:
@Tatsinnit resolving this comment and we can always come back and look at resolved one's. Thank you.